plyer/platforms/linux/sysinfo.py

```diff
@@ -0,0 +1,66 @@
+import platform
+import subprocess
+from subprocess import Popen, PIPE
+from plyer.facades import Sysinfo
+
+
+class LinuxSysinfo(Sysinfo):
+
+    def _model_info(self, **kwargs):
+        password = kwargs.get('password')
+        command = 'dmidecode -s baseboard-product-name'.split()
+        p = Popen(['sudo', '-S'] + command, stdin=PIPE, stderr=PIPE,
```

Comment:
You shouldn't be calling `sudo` here. The user's password won't be available to the app dev, and shouldn't be required to be entered into the program. You can get this information without root if you don't use dmidecode (and dmidecode may not be installed anyway). Also, `baseboard-product-name` isn't what you want here, but probably `system-product-name`.

```
ryan@ryan-mk3:~$ sudo dmidecode -s baseboard-product-name
1854
ryan@ryan-mk3:~$ sudo dmidecode -s system-product-name
HP 2000 Notebook PC
ryan@ryan-mk3:~$ cat /sys/devices/virtual/dmi/id/product_name 
HP 2000 Notebook PC
ryan@ryan-mk3:~$ sudo dmidecode -s system-manufacturer
Hewlett-Packard
ryan@ryan-mk3:~$ cat /sys/devices/virtual/dmi/id/sys_vendor 
Hewlett-Packard
```